api/v1alpha1/groupversion_info.go

```diff
@@ -28,7 +28,7 @@ var (
 	// GroupVersion is group version used to register these objects
 	GroupVersion = schema.GroupVersion{Group: "oadp.openshift.io", Version: "v1alpha1"}
 
-	Kind = "Velero"
```

Comment:
Should this be v1alpha2? A new group version.

Reply:
technically its a new API... so unsure if it matters